### DISCUSSION TITLE
Can't run functional test

### DISCUSSION THREAD

**Discussion by jsebvel** [adonisjs/core#2218] [Help]
I want to use functional test on my applications but I get one error when a use assert.JSONSubsect()
I'm creating an entity with attributes `document, name, email, phone, address,added_by`
this is the model created by Factory
![image](https://user-images.githubusercontent.com/38327125/107108850-436ad000-6809-11eb-98e1-4e9a1c79fdf0.png)

When Http post run it's ok, the data is on db and the response status is 200.
But in assert.JSONSubsect() I get this:

and this is the error, generated with timestamps and auto increments id
![image](https://user-images.githubusercontent.com/38327125/107108858-58dffa00-6809-11eb-8692-bab91f80c1c1.png)

I supose JSONSubsect does not need all the entity, and I use it beacouse official documentation say "This assertion tests a subset of objects, which is helpful when values inside an object are not determinable (e.g. timestamps).". And i supuse subset doesn't need all ansewer fields  just a fragment. But I don't know why this error.

this is the assertion fragment
![image](https://user-images.githubusercontent.com/38327125/107108974-529e4d80-680a-11eb-92bb-17420f28af34.png)

**Comment by thetutlage** [comment]
Not sure where exactly in the docs you have found `JSONSubsect` method. I am only aware of `response.assertJSONSubset`
